UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs

```diff
@@ -92,7 +94,15 @@ public bool WillInteract(MouseDrop interaction, NetworkSide side)
 		else
 		{
 			//trying to view this storage, can only drop on ourselves to view it
-			return interaction.Performer == interaction.TargetObject;
+			if (interaction.Performer != interaction.TargetObject) return false;
+			//if we're dragging another player to us, it's only allowed if the other player is downed
+			if (Validations.HasComponent<PlayerScript>(interaction.DroppedObject))
+			{
+				//dragging a player, can only do this if they are down / dead
+				return Validations.IsDeadOrCrit(interaction.DroppedObject, side);
```

Comment:
how about also checking for "stunned" (`registerPlayer.IsSlippingServer`) 
and "cuffed" (apparently `playerMove.IsCuffed`)?